requests-for-comments/2023-01-12-plan-naming.md

```diff
@@ -0,0 +1,47 @@
+# Request for comments: Renaming our plans by @raquelmsmith
+
+## Problem statement
+_Who are we building for, what are their needs (include example cases), why is this important?_
+
+Plan names are something seemingly minor that actually get used quite a lot, both internally and by our customers (oftentimes just inside their own heads when evaluating). However, our current plan names leave room for confusion:
+
+* The names don't indicate right at the get-go which plan you are probably interested in based on your company size/scale/needs. 
+* The names can include information about deployment but lack any real plan name (eg. PostHog Cloud is a paid tier but it's also a deployment)
+* We use the term self-serve in the docs but no one really knows what this means
+
+## Proposal
+The Cloud environment was new a couple years ago, so it made sense to call the plans on Cloud the "Cloud" plans. However, the Cloud environment is by far the default way to use PostHog now, and calling the plans "Cloud" is confusing (I can imagine thinking, "should I be here? or is there a different non-cloud thing I should use?").
+
+So, I'd propose that our Cloud plans are the _default_ plans and are named:
+
+- **Starter**
```

Comment:
If the user doesn't have a card in, are they on a plan? The hardest philosophical question ever 😅